packages/builder/src/components/automation/SetupPanel/AutomationBindingPanel.svelte

```diff
@@ -0,0 +1,202 @@
+<script>
+    import groupBy from "lodash/fp/groupBy"
+    import { Input, TextArea, Heading, Spacer, Label } from "@budibase/bbui"
+    import { createEventDispatcher } from "svelte"
+    import { isValid } from "@budibase/string-templates"
+    import {
+      getBindableProperties,
+      readableToRuntimeBinding,
+    } from "builderStore/dataBinding"
+    import { currentAsset, store } from "../../../builderStore"
+    import { handlebarsCompletions } from "constants/completions"
+  
+    const dispatch = createEventDispatcher()
+  
+    export let bindableProperties
+    export let value = ""
+    export let bindingDrawer
+    export let valid = true
+    export let bindings = []
+  
+    let originalValue = value
+    let helpers = handlebarsCompletions()
+    let getCaretPosition
+    let search = ""
+    let validity = true
+  
+    $: categories = Object.entries(groupBy("category", bindings))
+    $: value && checkValid()
+    $: bindableProperties = getBindableProperties(
```

Comment:
I don't think we need client app bindable properties here since we're using the different set of automation bindings instead.